roles/lib_openshift/src/test/unit/test_oadm_manage_node.py

```diff
@@ -179,6 +180,114 @@ def test_schedulable_false(self, mock_openshift_cmd, mock_tmpfile_copy):
         self.assertEqual(results['results']['nodes'][0]['name'], 'ip-172-31-49-140.ec2.internal')
         self.assertEqual(results['results']['nodes'][0]['schedulable'], False)
 
+    @unittest.skipIf(six.PY3, 'py2 test only')
+    @mock.patch('os.path.exists')
+    @mock.patch('os.environ.get')
+    def test_binary_lookup_fallback(self, mock_env_get, mock_path_exists):
+        ''' Testing binary lookup fallback '''
+
+        mock_env_get.side_effect = lambda _v, _d: ''
+
+        mock_path_exists.side_effect = lambda _: False
+
+        self.assertEqual(ManageNode._locate_oc_binary(), 'oc')
```

Comment:
I think these tests miss the fact that `oc` might be available in multiple directories in PATH, and the order is important, we should get the first.